mappings/net/minecraft/block/FruityLeavesBlock.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_2148 net/minecraft/block/FruityLeavesBlock
```

Comment:
Why did you decide on "fruity" for the name?

Reply:
Leaves of this type can drop apples.

Reply:
I personally think `AppleLeafBlock` would be better.
- use apple instead of fruity since it is more clear what it is meant to do
- use leaf singular since all blocks are singlular in `net.minecraft.block` (e.g.`AirBlock`, `BeaconBlock` etc.)

Reply:
1. Good idea
2. I used plural because the block id is also plural "leaves".